www/package.json

```diff
@@ -46,6 +46,7 @@
     "react": "18.0.0",
     "react-dom": "18.0.0",
     "react-typist": "^2.0.5",
+    "rehype-external-links": "^2.0.1",
```

Comment:
i think it wasn't removed from the source package.json?

Reply:
sorry forgot to commit that